config/locales/cy.yml

```diff
@@ -977,6 +977,143 @@ materials:
 
     <p>Os oes gennych unrhyw gwestiynau ynglŷn â'ch archeb, cysylltwch â ni yn <a href="mailto:brandio@cronfagymunedolylg.org.uk">brandio@cronfagymunedolylg.org.uk</a>.</p>
+user:
+  common:
+    yourAccount: Eich cyfrif
+    emailAddress: Cyfeiriad e-bost
+    password: Cyfrinair
+    passwordConfirmation:
+      label: Cadarnhad cyfrinair
+      explanation: Ail-deipiwch eich cyfrinair
+    passwordOld: Eich hen gyfrinair
+    passwordNew: Eich cyfrinair newydd
+    passwordRequirements: >
+      <p>Rhaid bod o leiaf 10 nod o hyd <br>
+      Rhaid bod yn wahanol i’ch cyfeiriad e-bost<br>
+      <a href="https://www.ncsc.gov.uk/collection/top-tips-for-staying-secure-online/use-a-strong-and-separate-password-for-email">
+        Darllenwch awgrymiadau ar sut i greu cyfrinair cryf
+        <span class="u-visually-hidden"> National Cyber Security  - Dolen allanol</span>
+      </a></p>
+    loggedInAs: Wedi mewngofnodi fel
+    logOut: allgofnodi
+    alertMessages:
+      activationSent: Mae e-bost wedi ei anfon i’ch cyfeiriad e-bost cofrestredig gyda chyfarwyddiadau i actifadu eich cyfrif.
+      activationComplete: Mae eich cyfrif wedi ei actifadu’n llwyddiannus!
+      passwordUpdated: Mae eich cyfrinair wedi ei ddiweddaru’n llwyddiannus!
+      emailUpdated: Mae eich cyfeiriad e-bost wedi ei ddiweddaru’n llwyddiannus!
+      loggedOut: Rydych wedi allgofnodi’n llwyddiannus.
+      passwordResetRequest: Cais am ail-osod cyfrinair. Os yw’r cyfeiriad e-bost yn gywir, byddwch yn derbyn cyfarwyddiadau pellach dros e-bost.
+    activationRequired: >
+      Nawr, rydym angen sicrhau fod y cyfeiriad e-bost rydych newydd ei ddarparu yn gywir. Gwiriwch eich e-byst i actifadu eich cyfrif. Neu a oes angen i ni <a href="/user/activate">ail-anfon y ddolen actifadu</a>?
+  validationMessages:
+    emailInvalid: Rhowch gyfeiriad e-bost dilys
+    oldPasswordRequired: Rhowch eich cyfrinair presennol
+    oldPasswordWrong: Roedd eich hen gyfrinair yn anghywir
+    passwordRequired: Rhowch eich cyfrinair
+    passwordMatchesEmail: Rhaid i’r cyfrinair fod yn wahanol i’ch cyfeiriad e-bost
+    passwordLength: Rhaid i’r cyfrinair fod o leiaf %s nod o hyd
+    passwordStrength: Cyfrinair yn rhy wan, ceisiwch gyfrinair gwahanol
+    passwordConfirmation: Rhaid i’r cyfrineiriau gyfateb
+  dashboard:
+    title: Eich cyfrif
+    sectionLinks:
+    - label: Newid eich cyfrinair
+      href: /user/password/reset
+    - label: Newid eich cyfeiriad e-bost
+      href: /user/update-email
+    - label: Gweld eich ceisiadau ariannu
+      href: /apply/awards-for-all
+  login:
+    title: Mewngofnodi
+    introduction: >
+      <p>Mae hwn yn ffurflen gais newydd. Os nad ydych wedi mewngofnodi yma eisoes, efallai na fydd gennych gyfrif â ni - a bydd angen <a href="/user/register">creu cyfrif newydd</a> i ymgeisio.</p>
```

Comment:
Since, the hyperlinks for this sentence was not underlined, I online translated and replaced the hyperlink text. Hence, might need reviewing.

Reply:
Oooh. Yeah should avoid doing this, mind switching them back to english and we can send Ossian a list of stuff that is untranslated.

Reply:
Well, it was translated but the hyperlinks were not underlined in the Welsh sentence. Should I revert it back in that case ?

Reply:
oh! nope this is fine :)

Reply:
Sorry, my bad. That comment didn't explain it very well :)